src/app/components/ui/ToggleBoolean.js

```diff
@@ -0,0 +1,16 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
I don't see any advantage of using this component rather than `<Toggle>`. If you really want to use it, please put it in `react-components` project instead of here.

Reply:
![output](https://user-images.githubusercontent.com/713283/56123165-331e9c80-5f74-11e9-85ea-12b6514e963a.gif)

1. ToggleBoolean 
2. Toggle

Why ? Because of the UX

Reply:
`Toggle` [has now a loading state](https://github.com/ProtonMail/react-components/commit/f1a75668987bc839a1cc0a0147e3e145e785b8c4) so you can use it when a promise is running.